gitlint/cli.py

```diff
@@ -117,8 +118,16 @@ def lint(ctx):
 
     # Let's get linting!
     linter = GitLinter(lint_config)
-    violations = linter.lint(last_commit)
-    linter.print_violations(violations)
+    number_of_commits = len(gitcontext.commits)
+    for index, commit in enumerate(gitcontext.commits):
+        violations = linter.lint(commit)
+        if violations:
+            click.echo(u"{0}Commit {1}:".format(
```

Comment:
I think we want to add an extra check here where we only print "Commit x" in case more than 1 commit has been linted. This ensures backwards compatibility as well.